capistrano3-puma.gemspec

```diff
@@ -19,5 +19,4 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_dependency 'capistrano', '~> 3.0'
-  spec.add_dependency 'puma' , '>= 2.6'
```

Comment:
That a special case, most people have the configs in in their project. And some configs are only available after 2.6